src/lambda-helpers.ts

```diff
@@ -47,8 +47,8 @@ export const getGithubOpts = () => ({
 const twitterConsumerKey = process.env.TWITTER_CONSUMER_KEY || "";
 const twitterConsumerSecret = process.env.TWITTER_CONSUMER_SECRET || "";
 
-export const getTwitterOpts = () => {
-  const twitterBearerTokenResponse = wrapAxios(
+export const getTwitterOpts = async () => {
+  const twitterBearerTokenResponse = await wrapAxios(
```

Comment:
cc @makalaaneesh 

`wrapAxios` is `async`, so we need to await the result before retrieving the body. no worries, just an fyi 😄

Reply:
Can you share some article/video to get a basic understanding of the whole async nature of node/js in general? 😄

Reply:
Hm I don't have one off the top of my head, I'd probably do some googling to find one. basically every async function you need to await it in order to get the result synchronously.

Maybe I'll write one!

Reply:
Yea, here's a good one https://levelup.gitconnected.com/async-await-vs-promises-4fe98d11038f